GNUmakefile

```diff
@@ -13,6 +13,8 @@ PKG_VERSION_PATH=github.com/jfrog/terraform-provider-${PRODUCT}/${PKG_NAME}
 VERSION := $(shell git tag --sort=-creatordate | head -1 | sed  -n 's/v\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1.\2.\3/p')
 NEXT_VERSION := $(shell echo ${VERSION}| awk -F '.' '{print $$1 "." $$2 "." $$3 +1 }' )
 BUILD_PATH=terraform.d/plugins/registry.terraform.io/jfrog/${PRODUCT}/${NEXT_VERSION}/${TARGET_ARCH}
+SONAR_SCANNER_VERSION?=4.7.0.2747
+SONAR_SCANNER_HOME?=$HOME/.sonar/sonar-scanner-$SONAR_SCANNER_VERSION-macosx
```

Comment:
macosx?

Reply:
Yes, for local execution. The pipeline step will override that with the linux version of the path

Reply:
https://github.com/jfrog/terraform-provider-artifactory/pull/650/files#diff-99b4c1d653c1f12db90b59cb459ad016aa08efeced1acdbe0cf70a52511dd867R41

Reply:
yeah, I see now

Reply:
TBH I don't think the path needs the OS in the suffix at all